stdsimd-test/assert-instr-macro/Cargo.toml

```diff
@@ -5,3 +5,9 @@ authors = ["Alex Crichton <alex@alexcrichton.com>"]
 
 [lib]
 proc-macro = true
+
+[dependencies]
+proc-macro2 = { version = "0.1", features = ["unstable"] }
+quote = { git = 'https://github.com/dtolnay/quote' }
+syn = { git = 'https://github.com/dtolnay/syn', features = ["full"] }
+synom = { git = 'https://github.com/dtolnay/syn' }
```

Comment:
If/when this crate gets moved to `core`, will we still be able to carry these dependencies?

Reply:
Orthogonally, is there a way to use non-git dependencies? I actually kind of want it to be possible for `stdsimd` to be on crates.io. It would be nice to use it before it gets to `core`. :-)

(I could very well be misunderstanding. Do these need to be non-git deps to be able to put `stdsimd` on crates.io?)

Reply:
Yeah I think this'll work out just fine, even when this is moved into `core`. The `core` crate itself isn't actually tested, and my guess is we'll still have this repo *actually* doing all the testing and then in-tree we won't do any testing. These dependencies and such shouldn't harm the build in rust-alng/rust though.


Also, yeah, it's true that this'd hinder publishing on crates.io. What we'd just do though is comment out the `[dev-dependencies]` section of Cargo.toml in the `stdsimd` crate, and then we'd be good to go.

Reply:
Bummer, but all right, that's fine.